src/backend/sgx/attestation.rs

```diff
@@ -434,6 +442,62 @@ pub fn get_quote(report: &[u8], akid: Vec<u8>, out_buf: &mut [u8]) -> Result<usi
     Ok(quote.len())
 }
 
+/// Gets quote size with CRL added
+pub fn get_quote_size_with_crl(akid: Vec<u8>) -> Result<usize, Error> {
+    let q_size = get_quote_size(akid.clone())?;
+    let mut out_buf_temp = Vec::with_capacity(q_size);
+    let report = [0u8; SGX_REPORT_SIZE];
+    get_quote(&report, akid, &mut out_buf_temp)?;
+
+    let mut crl_file = sgx_cache_dir()
+        .map_err(|e| Error::new(ErrorKind::Other, format!("SGX CRL read error {}", e)))?;
+    crl_file.push("crls.der");
+
+    let evidence = SgxEvidence {
```

Comment:
@haraldh @dpal what's better, a `struct` that we convert to DER, or just add the CRL bytes to the end of the report bytes? Just adding the bytes might be easier, since the CRL bytes are already DER-encoded, and saves the issue ot figuring how how big the report is once made into DER.

Secondly, the `exec-wasmtime` already converts the report to ASN1 when making the extension, so we probably don't want two levels of ASN1 encoding.

The only issue is making sure we get the remaining (CRL) bytes after Rust converts the bytes to the C-style struct.